examples/introduction_pytorch_poutyne.ipynb

```diff
@@ -29,7 +29,7 @@
     "from torchvision import transforms, utils\n",
     "from torchvision.datasets.mnist import MNIST\n",
     "\n",
-    "from poutyne.framework import Model, ModelCheckpoint, CSVLogger, Callback\n",
+    "from poutyne.framework import Model, ModelCheckpoint, CSVLogger, Callback, Experiment\n",
     "from poutyne import torch_to_numpy"
```

Comment:
We should call the new function in util `set_seed`.

Reply:
Done in cd0cdb9